docs/blog/20/5-gettig_started_to_vitis_acceleration_flow_with_zynq_7000.md

```diff
@@ -0,0 +1,351 @@
+SUPPORT_URL="http://help.ubuntu.com/"
+BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
+VERSION_CODENAME=xenial
+UBUNTU_CODENAME=xenial
+```
+
+To trick the installer, set your OS version to one of the closest supported one
+with your OS. Open with the text editor with command below and change `16.04.7`
+to  `16.04.6`.
+
+```console
+$ sudo gedit /etc/os-release
+```
+
+Now, make sure also that you have required software packages or run the command below
+to install them.
+
+```console
+$ sudo add-apt-repository ppa:xorg-edgers/ppa sudo apt-get updatesudo apt-get install libgl1-mesa-glxsudo apt-get install libgl1-mesa-drisudo apt-get install libgl1-mesa-devsudo add-apt-repository --remove ppa:xorg-edgers/ppasudo apt install net-toolssudo apt-get install -y unzipsudo apt install gccsudo apt install g++sudo apt install pythonln -s /usr/bin/python2 /usr/bin/python
+```
+
+You can now run the installer in the following step.
+
+### 1. Vitis
+
+Go Xilinx download page <https://www.xilinx.com/support/download.html>
+
+I prefer downloading **Linux Self Extracting Web Installer**.
+
+![Vitis_Download](img/5-vitis_download.png)
+
+Make the bin file executable `$ chmod +x Xilinx_Unified_2020.1_0602_1208_Lin64.bin`
+and run the installer `$ ./Xilinx_Unified_2020.1_0602_1208_Lin64.bin`.
+
+Follow the installation defaults. After the installation; Vivado, Vitis and other
+xilinx tools should be under `/opt/tools/Xilinx/` directory.
+
+### 2. Xilinx Runtime(XRT)
+
+Go Xilinx download page <https://www.xilinx.com/support/download.html> and open
+`Vitis Embedded Platforms` tab.
+
+![XRT_Download](img/5-xrt_download.png)
+
+Download `XRT 2020.1` and install XRT with double click to `.deb` file or run
+the installation command `$ sudo apt install <filename>.deb`. Now, you should
+see `/opt/xilinx/xrt/` directory created and **XRT** is installed.
+
+### 3. Embedded Platform Files for ZC706
+
+Go Xilinx download page <https://www.xilinx.com/support/download.html> and open
+`Vitis Embedded Platforms` tab again.
+
+![Platform_Download](img/5-platform_download.png)
+
+Download ZC706 base platform zip file. Extract the zip file to location `/opt/xilinx/platforms/`.
+Now, you should have file `xilinx_zc706_base_202010_1` under this directory.
+
+### 4. Prebuilt Petalinux Kernel and Root File System
+
+We will use prebuilt kernel and rootfs for simplicity. Go Xilinx download page
+<https://www.xilinx.com/support/download.html> and open `Petalinux` tab.
+
+![Common_Download](img/5-zynq_common_download.png)
+
+Download `ZYNQ Common Image` and extract it to `/opt/tools/Xilinx/xilinx-zynq-common-v2020.1/`.
+Run `sh sdk.sh` script again in this folder. This script should create **sysroot**
+under `/opt/petalinux/2020.1/`.
+
+### 5. Tutorial Files
+
+In your home directory create `projects` folder to download tutorial files there.
+
+* Within `projects`, clone tutorial files from github repository `$ git clone https://github.com/aniltirli/Vitis-Tutorials.git`
+
```

Comment:
Burada neden senden clone'luyoruz :smiley: `patch-1` in bir anlamı var mı?

Reply:
normalde zc706 yok, zc706 için eklemeler yaptım.